packages/marketplace/src/actions/developer.ts

```diff
@@ -18,3 +19,8 @@ export const setMyIdentity = actionCreator<DeveloperModel>(ActionTypes.DEVELOPER
 export const fetchBilling = actionCreator<FetchBillingParams>(ActionTypes.DEVELOPER_FETCH_BILLING)
 export const fetchBillingSuccess = actionCreator<Billing>(ActionTypes.DEVELOPER_FETCH_BILLING_SUCCESS)
 export const fetchBillingFailure = actionCreator<unknown>(ActionTypes.DEVELOPER_FETCH_BILLING_FAILURE)
+export const fetchMonthlyBilling = actionCreator<FetchMonthlyBillingParams>(ActionTypes.DEVELOPER_FETCH_MONTHLY_BILLING)
+export const fetchMonthlyBillingSuccess = actionCreator<MonthlyBilling>(
+  ActionTypes.DEVELOPER_FETCH_MONTHLY_BILLING_SUCCESS,
+)
+export const fetchMonthlyBillingFailure = actionCreator<unknown>(ActionTypes.DEVELOPER_FETCH_MONTHLY_BILLING_FAILURE)
```

Comment:
Please change `DEVELOPER_FETCH_MONTHLY_BILLING_FAILURE` to `DEVELOPER_FETCH_MONTHLY_BILLING_FAILED`